SPECS/inih/inih.spec

```diff
@@ -0,0 +1,134 @@
+Summary:  A Simple INI file parser library
+Name:     inih
+Version:  57
+Release:  1%{?dist}
+License:  BSD
+URL:      https://github.com/benhoyt/inih
+Source0:  https://github.com/benhoyt/inih/archive/refs/tags/inih-r57.tar.gz
```

Comment:
```suggestion
#Source0: https://github.com/benhoyt/inih/archive/refs/tags/r57.tar.gz
Source0:   inih-r57.tar.gz
```

Reply:
This is a bit of a hack to better support renaming source tar.gz etc. with bad names. Our internal tools will accept either `#Source0` or `Source0` as valid options when validating manifests etc. In this case we DO want the source file to be `inih-*.tar.gz` with the package name included, but the actual valid URL remains with the version only.

Reply:
Oh, i did not know this!